ci/requirements-2.7-0.19.2.pip

```diff
@@ -1,5 +1,7 @@
-httplib2
-google-api-python-client==1.2
-python-gflags==2.0
-oauth2client==1.5.0
+google-api-python-client
```

Comment:
can you fix one of these builts to the oldest versions that are possible to make this work (of google-api-python-client at least)